### DISCUSSION TITLE
i cant clone.  Windows 7.  Tried several times.   Thanks!  Error in message

### DISCUSSION THREAD

**Discussion by ghost** [community/community#21544] [New to GitHub]
<p>I keep getting this error. Thanks for looking!</p>
<p>I am on windows 7 laptop.</p>
<p>I am using </p>
<p>Cloning into ‘C:\Users\857723\Documents\GitHub\speed-limit-grabber-chrome-extension’… fatal: unable to access ‘<a href="https://github.com/carterson2/speed-limit-grabber-chrome-extension.git/" rel="nofollow noopener">https://github.com/carterson2/speed-limit-grabber-chrome-extension.git/</a>’: schannel: next InitializeSecurityContext failed: Unknown error (0x80092013) - The revocation function was unable to check revocation because the revocation server was offline. Error(s) during clone: git clone failed: exit status 128</p>

**Comment by jforoughi** [comment]
<p>From the looks of the error message, my guess would be something related to your proxy settings (if you have one) or possibly an untrusted certificate error, which is making GitHub act weary about it. </p>
<p>If you have another computer, I’d suggest trying to clone that repo and see if you’re still getting that error message from that PC.</p>

**Comment by ghost** [comment]
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/3/3b95520bdd9dab77b529bedd8c495921c219d199.png" data-download-href="/uploads/short-url/8v65tXmTo36xNuTHstjbW3nXaSB.png?dl=1" title="Untitled.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/10137/181065385-fe1ac56b-d46e-4b75-9db6-b12e8e9270a6.png" alt="Untitled.png" data-base62-sha1="8v65tXmTo36xNuTHstjbW3nXaSB" width="690" height="80" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/3/3b95520bdd9dab77b529bedd8c495921c219d199_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Untitled.png</span><span class="informations">1074×125 15.2 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>I have mr Gonzalas at github support looking at this.</p>
<p>I forgot that our company doesn’t like github…</p>
<p>ugh</p>

**Comment by jforoughi** [comment, marked as answer]
<p>Gotta love work. Anyway, I’m glad you found what was causing the issue. If your company still gives you issues, I’d say just clone the repo on a home computer and either put it on a flash drive or in the cloud (assuming all you wanted to do was download the repo listed in the OP)</p>
